src/clocks/digital-clock.py

```diff
@@ -25,10 +30,21 @@ def main():
 if __name__ == "__main__":
     root = Tk()
     root.title("Digital Clock")
+    root.geometry("370x100")
+    root.resizable(0, 0)
```

Comment:
calling this root can be confusing with other concepts where root is specifically used. Since Tkinter is Python's binding to the Tk's GUI toolkit, which creates a window pop out display, it should aptly be named as app_window.